scripts/jest/ts-preprocessor.js

```diff
@@ -28,7 +28,7 @@ function compile(content, contentFilename) {
       // the file path into backslashes on Windows.
       filename = path.normalize(filename);
       var reactRegex = new RegExp(
-        path.join('/', '(?:React|ReactDOM)(?:\.d)?\.ts$')
+        '\\' + path.join('/', '(?:React|ReactDOM)(?:\.d)?\.ts$')
```

Comment:
Can you explain why this helps exactly? There is a comment right above:

```js
 
       // `path.normalize` and `path.join` are used to turn forward slashes in
        // the file path into backslashes on Windows.
```

Are they not helping? Why?

Ideally I’d like to see us use one separator style, and normalize paths when necessary.